ASTRO/ASTROpdd.tex

```diff
@@ -0,0 +1,264 @@
+      % Consider the abstract to be your elevator pitch to anyone reading this design document.
+      % What are they reading?
+      % What is the goal?
+      % Why is it worth my time?
+      % The abstract is what will show up in Google results and other search engines, and what people will read when they are deciding what is worth their time and brain power.
+\end{abstract}
+
+\label{sec:nomenclature}
+\newcommand{\nomunit}[1]{%
+\renewcommand{\nomentryend}{\hspace*{\fill}#1}}
+\renewcommand{\nompreamble}{
+    % If you include mathematical expressions or express variables in the design document, list them with their corresponding definitions here as a list.
+    % The two lines below make it look nice when defining units/values to constants.
+
+    % Note that math terms and non-math terms are separated and alphabetized, regardless of the order in which they are defined. (Recall terms \$like this\$ are in the math environment)
+    % Read more about advanced nomenclature formatting here:\\
+    % \url{https://www.sharelatex.com/learn/Nomenclatures}
+  }
+\nomenclature{RIT}{Rochester Institute of Technology}
+\nomenclature{SPEX}{RIT Space Exploration}
+\nomenclature{PDD}{Project Design Document}
+% Below are examples of using nomenclature for math symbols and constants or units
+
+
+% HELPFUL HINT: If you get the warning ``Command terminated with space.'' when using a \command try placing ``%'' or ``{}'' immediately following the command.
+
+% The sections included here are required. Additional sections and subsections may be added as necessary.
+\section{Introduction}
+\label{sec:introduction}
+  % The introduction is a place to give background and context before diving into the subject matter.
+  % Establish context for the work you are about to propose and the main ideas of the proposition itself.
+
+\IEEEPARstart{T}{he} RIT Astro Tracking team formed in the Fall of 2016 and was designed for the purpose of Astronomical research and the tracking of artificial satellites.  Considerable growth has been made in regard to general Astronomy skills with a number of quality images and the improvement of general astrophotography skills.  Clear pictures of Jupiter, Saturn, and other objects have been taken by members and showcased at various SPEX related events.  In addition the team has developed software for the automation of its primary telescope, the Meade LX200, where it can be controlled from any location with wireless connectivity.
+
+The team has accomplished much since its inception and the next logical step would be to improve the skills developed as well as begin preliminary tracking of both artificial satellites and other celestial objects.  In order to begin new members will firstly have to be educated on basics, technology to assist in tracking must be developed, and technology must be developed for the maintenance of current equipment.
+
+
+\section{Primary Objective}
```

Comment:
Six primary objectives are listed here. Are they all of equal priority? 
* If yes, no changes needed. 
* If not, it would be useful to clarify that somehow.

Reply:
Yes, all of equal priority.

Reply:
:white_check_mark: